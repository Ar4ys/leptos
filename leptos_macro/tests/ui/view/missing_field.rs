use leptos::*;

#[component]
fn Component(prop: i32) -> impl IntoView {
    _ = prop;
}

fn main() {
    view! {
        <Component />
    };
}
