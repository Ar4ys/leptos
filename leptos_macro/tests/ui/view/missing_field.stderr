warning: use of deprecated method `ComponentPropsBuilder::build`: Missing required field prop
  --> tests/ui/view/missing_field.rs:10:10
   |
10 |         <Component />
   |          ^^^^^^^^^
   |
   = note: `#[warn(deprecated)]` on by default

error[E0061]: this method takes 1 argument but 0 arguments were supplied
  --> tests/ui/view/missing_field.rs:10:10
   |
10 |         <Component />
   |          ^^^^^^^^^ an argument of type `ComponentPropsBuilder_Error_Missing_required_field_prop` is missing
   |
note: method defined here
  --> tests/ui/view/missing_field.rs:3:1
   |
3  | #[component]
   | ^^^^^^^^^^^^
   = note: this error originates in the derive macro `::leptos::typed_builder_macro::TypedBuilder` (in Nightly builds, run with -Z macro-backtrace for more info)
help: provide the argument
   |
10 |         <Component(/* ComponentPropsBuilder_Error_Missing_required_field_prop */) />
   |                   +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
